src/ui/widgets/calendar.rs

```diff
@@ -0,0 +1,111 @@
+use super::*;
+
+const DAY_SIZE: (u16, u16) = (4, 3);
```

Comment:
what is 4 what is 3? I guess its:
const DAY_WIDTH: u16 = 4;
const DAY_HEIGHT: u16 = 3;

Reply:
double comment